api.go

```diff
@@ -84,8 +84,9 @@ type (
 	}
 
 	createStreamReq struct {
-		Name    string   `json:"name,omitempty"`
-		Presets []string `json:"presets,omitempty"`
+		Name     string   `json:"name,omitempty"`
+		ParentID string   `json:"parentId,omitempty"`
```

Comment:
I think this `ParentId` field is not being used 👀 

But I think it is being used in stream-tester right? Maybe we should move that change here as well (incl that better `CreateStream` abstraction, if I remember it correctly) so it's easier to migrate stream-tester to this soon